design/roadmap.md

```diff
@@ -1,179 +1,71 @@
-# ksonnet Roadmap
+# Roadmap to 1.0
 
-This document captures open questions and features for ksonnet.
-*Note that the ordering of items is unrelated to their priority or order of completion.*
+## Using ksonnet shouldn't require jsonnet knowledge
 
-Improvements are planned in the following areas:
+* Users can initialize a Ksonnet project, add components, configure parameters, customize their environment, and apply configurations without knowledge of Jsonnet.
 
-* Dependency management
-   * [Support third-party registries (beyond Github protocol)](#support-third-party-registries-beyond-github-protocol)
-   * [Easily generate third-party registries](#easily-generate-third-party-registries)
-   * [Lazily download and update dependencies (`ks install`)](#lazily-download-and-update-dependencies-ks-install)
-* Component workflow
-   * [Expose "environment metadata" to prototypes](#expose-environment-metadata-to-prototypes)
-   * [Support JSON and YAML](#support-json-and-yaml)
-   * [Reference components in other components](#reference-components-in-other-components)
-   * [Establish clear `ks apply` semantics](#establish-clear-ks-apply-semantics)
-   * [Cleanly remove components](#cleanly-remove-components)   
-   * [Encryption support for secrets](#encryption-support-for-secrets)
-* Development tooling
-   * [Improve output of `ks diff`](#improve-output-of-ks-diff)
-   * [Show errors in app files (`ks lint`)](#show-errors-in-app-files-ks-lint)
-   * [Update VSCode extension](#update-vscode-extension)
-* Under the hood
-  * [Improve `ksonnet-lib` generation](#improve-ksonnet-lib-generation)
-  * [Refactor common code into `ksonnet/client`](#refactor-common-code-into-ksonnetclient)
+## Component namespaces
 
+* Users can segregate their components into namespaces for more convenient organization.
```

Comment:
Do you think the overloading of "namespaces" might cause confusion? 
Are component namespaces implicitly tied to hierarchical directory structure? 

If so, consider the following (feel free to only take the bits you like):
```
## More precise component usage
* Users can organize their components hierarchically, using directory structure
* Users can customize an environment to target a specific subset of components
```